src/main/kotlin/com/workos/webhooks/WebhooksApi.kt

```diff
@@ -40,7 +40,7 @@ class WebhooksApi() {
     }
   }
 
-  private fun verifyHeader(
+  fun verifyHeader(
```

Comment:
Can we add comments to these methods now that they are public? Similar to this:

https://github.com/workos/workos-kotlin/blob/ff3b370c5327ce4ce57cd427bdbc8acf0e6e05bd/src/main/kotlin/com/workos/webhooks/WebhooksApi.kt#L19

Reply:
Added comments describing the method params.